editor.planx.uk/src/@planx/components/List/schemas/Trees.ts

```diff
@@ -16,6 +16,7 @@ export const Trees: Schema = {
       type: "text",
       data: {
         title: "Proposed work",
+        description: "This is the wee descy",
```

Comment:
This schema is in use in the "real" flows that we're user-testing - we don't want to add this fake `description` directly here? Please see `List` component schemas for examples of how we keep separate mock schemas for testing purposes.